### PR TITLE
cleaning up some things with status page

### PR DESCRIPTION
**Pull request by s-laugh**
### Description

List of proposed changes:

- Enable a Reset to bring you back to landing on success getting a status
- Convert page to be static
- Remove some un-used code
- fix the cron

### What to test for/How to test

If you get a status response, the "go back" should be "restart" and bring you back to the landing page to start the flow again.

### CODE REVIEW

File: pages/status.tsx [hunk 1]
```diff
@@ -67,13 +73,13 @@ const Status: FC = () => {
...
   }, [formik, t])
 
-  const handleReset: MouseEventHandler<HTMLButtonElement> = useCallback(
-    (e) => {
-      e.preventDefault()
-      formik.resetForm()
-    },
-    [formik]
-  )
+  // const handleReset: MouseEventHandler<HTMLButtonElement> = useCallback(
```
Comment (sebastien-comeau) [inline pages/status.tsx:76]: This commented block of code should be removed. We can retreive it in previous commits if needed.

File: pages/status.tsx [hunk 2]
```diff
@@ -93,14 +99,16 @@ const Status: FC = () => {
       {}
       {checkStatusReponse !== undefined ? (
         <StatusInfo
-          handleGoBackClick={handleGoBack}
-          goBackText={t('go-back')}
+          handleGoBackClick={
+            checkStatusReponse ? () => Router.push('/landing') : handleGoBack
```
Comment (sebastien-comeau) [inline pages/status.tsx:103]: Move this if condition inside `handleGoBack` and set `onGoBackClick` prop with `handleGoBack` as before. I think we should rename `handleGoBackClick` to `onGoBackClick` to be consistent with other event props naming convention.

File: pages/status.tsx [hunk 3]
```diff
@@ -93,14 +99,16 @@ const Status: FC = () => {
       {}
```
Comment (sebastien-comeau) [inline pages/status.tsx:99]: Remove this empty JSX curly braces

File: pages/status.tsx (hunk 1 above)
Comment (sebastien-comeau) [inline pages/status.tsx:76]: This commented block of code should be removed. We can retreive it in previous commits if needed.

File: pages/status.tsx (hunk 3 above)
Comment (sebastien-comeau) [inline pages/status.tsx:99]: Remove this empty JSX curly braces

File: pages/status.tsx (hunk 2 above)
Comment (sebastien-comeau) [inline pages/status.tsx:103]: I think moving this if condition inside `handleGoBack` and set `onGoBackClick` prop with `handleGoBack` as before would make more sense.

We should also rename `handleGoBackClick` to `onGoBackClick` to be consistent with other event property naming convention, as example `onClick`, `onChange`, etc...

File: pages/status.tsx [hunk 4]
```diff
@@ -6,12 +6,18 @@ import ErrorSummary, {
...
+import { CheckStatusRequestBody } from './api/check-status'
 import { useCheckStatus } from '../hooks/api/useCheckStatus'
 import { useFormik } from 'formik'
 import * as Yup from 'yup'
 import useTranslation from 'next-translate/useTranslation'
 import StatusInfo from '../components/StatusInfo'
+import { GetStaticProps } from 'next'
+import Router from 'next/router'
+
+export const getStaticProps: GetStaticProps = async () => ({
```
Comment (sebastien-comeau) [inline pages/status.tsx:18]: I suggest to move `getStaticProps` before `export default Status` as you would see in next.js examples.

**Review by sebastien-comeau — CHANGES_REQUESTED** [review]

File: pages/status.tsx (hunk 4 above)
Comment (sebastien-comeau) [reply pages/status.tsx:18]: https://nextjs.org/docs/basic-features/data-fetching/get-static-props#using-getstaticprops-to-fetch-data-from-a-cms

File: pages/status.tsx (hunk 1 above)
Comment (s-laugh) [reply pages/status.tsx:76]: I'd rather leave it for now; we are getting requirements from BAs soon that will change some things.

File: pages/status.tsx (hunk 4 above)
Comment (s-laugh) [reply pages/status.tsx:18]: I honestly hate that that's how examples look; its so very backwards as this function is always called before the actual component. It's far more logical to keep it at the top.

File: pages/status.tsx (hunk 1 above)
Comment (sebastien-comeau) [reply pages/status.tsx:76]: Sounds good

File: pages/status.tsx (hunk 4 above)
Comment (sebastien-comeau) [reply pages/status.tsx:18]: I see your point but it's a convention that you will see everywhere with next.js. I like to follow next.js convention.

File: pages/status.tsx (hunk 2 above)
Comment (sebastien-comeau) [reply pages/status.tsx:102]: It's why I was using 2 different status component before. 1 for when you receive an actual status `StatusInfo` and one when status is null `StatusInfoUnavailable`. It made the code easier to follow.

File: pages/status.tsx (hunk 4 above)
Comment (s-laugh) [reply pages/status.tsx]: okay.... :(

File: pages/status.tsx (hunk 2 above)
Comment (s-laugh) [reply pages/status.tsx:98]: If your suggesting it goes back to a single condition now, I'm okay to do that.

Comment (sebastien-comeau) [reply pages/status.tsx:98]: For now, yes.

**Pull request merged**